src/commands/build/index.ts

```diff
@@ -84,6 +86,22 @@ export default class BuildCommand extends Command {
       releasePkg.typings = 'src/index.d.ts';
     }
 
+    // Convert any yalced dependencies (local "file:" dependencies) into proper bundledDependencies before publishing
+    for (const [name, version] of Object.entries(releasePkg.dependencies as { [key: string]: string })) {
```

Comment:
How about `Dictionary`?

Reply:
I tried that, and it didn't recognize it. So IDK maybe this project doesn't use `@stoplight/types` or something.

Reply:
p.s. Checked and yes, this package does not use `@stoplight/types`. This package doesn't use any `@stoplight/` packages at all. Which is probably a good thing, as it avoids any setting up any weird mutual dependency chains.